/**:
  ros__parameters:
    mcu_ip_addr: 192.168.9.5

/driver/sensors_esp32_driver/button_battery_state_component_node:
  ros__parameters:
    sub_port: 9000
    timeout_ms: 1000

/driver/sensors_esp32_driver/buzzer_switch_component_node:
  ros__parameters:
    mcu_port: 9001
    this_port: 9001

/driver/sensors_esp32_driver/depth_component_node:
  ros__parameters:
    this_port: 9002
    mcu_port: 9002
    timeout_ms: 500

/driver/sensors_esp32_driver/environment_component_node:
  ros__parameters:
    sub_port: 9003
    timeout_ms: 1000

/driver/sensors_esp32_driver/imu_reset_component_node:
  ros__parameters:
    mcu_port: 9004
    this_port: 9004

/driver/sensors_esp32_driver/leak_component_node:
  ros__parameters:
    sub_port: 9005
    timeout_ms: 5000

/driver/sensors_esp32_driver/rgb_component_node:
  ros__parameters:
    mcu_port: 9006
    this_port: 9006
 
/driver/sensors_esp32_driver/rtc_gnss_component_node:
  ros__parameters:
    sub_port: 9007
    timeout_ms: 1000

/driver/sensors_esp32_driver/rtc_time_component_node:
  ros__parameters:
    sub_port: 9008
    timeout_ms: 1000

/driver/sensors_esp32_driver/tilt_servo_component_node:
  ros__parameters:
    mcu_port: 9009
    this_port: 9009

/driver/sensors_esp32_driver/zed_power_component_node:
  ros__parameters:
    mcu_port: 9010
    this_port: 9010

/driver/sensors_esp32_driver/system_status_component_node:
  ros__parameters:
    sub_port: 9012
    timeout_ms: 500
